### PR TITLE
README update

### PR DESCRIPTION
**Pull request by nickmonad**

### CODE REVIEW

**Comment by nickmonad** [comment]
rendered version here: https://github.com/lighthouse/lighthouse/blob/readme-update/README.md

@lighthouse/owners Let me know if I missed any information we want here. The build steps listed seemed to work well for me but let me know if anything needs to be added for varying platforms/go installs/etc

**Comment by cmbrose** [comment]
+1 Looks awesome to me!

**Comment by theMagicalKarp** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -1,4 +1,41 @@
-docker-manager
+Lighthouse
 ==============
 
-Create and manage execution containers in the cloud
+[Lighthouse](https://lighthouse.github.io) is a Docker controller. It aggregates information about Docker instances arcoss mulitple cloud providers and allows for easy control over the containers deployed on that system.
```
Comment (zach-taylor) [inline README.md]: arcoss -> across

File: README.md [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+
+It bridges the gap between providers hosting docker services. Are you running containers accross hundreds of vms accross the world wide web? No problem, Lighthouse gives you the power to manage and maintain those vms with a few simple clicks. Everything was built with the goal of monotizing AWS, GCE, Azure, ect. into an easy to manage platform.
```
Comment (zach-taylor) [inline README.md]: accross -> across

Comment (robertsheehy-wf) [inline README.md]: My bad :sweat_smile:

**Comment by zach-taylor** [comment]
Looks good +1

**Pull request merged**